README.md

```diff
@@ -1,36 +1,40 @@
-# Python Project Template
+# Example Data Science Project
 
 ## Description
 
-This is a template repository that can be used for Python 3.11 projects and uses the Poetry package manager. By default this project sets up the following:
+This repository is a typical example of a data science project written in Python and is used for testing purposes when
+we need to have a more truthful example of an algorithm in use by the Dutch Government.
 
-* Devcontainers for VSCode users
-* Github community health files
-* Github Dependabot
-* VSCode configs
-* Some scripts to adhere to programming standards
-* A editorconfig file so editors enforce formatting
-* A default .gitgignore
-* A default pre-commit-config
-* A EUPL v1.2 Licence
-* A basic Docker setup
-* publiccode.yml
+The typical steps will be done:
 
-## How to use this template repository
+0. exploratory data analysis
+1. prepping data
+2. training models
+3. evaluating models
+4. serving a result
+5. monitoring
 
-When creating a new Repository select this template repository as the base.
+Because building an algorithm is not linear and if unwanted behaviour is seen in any step of the progress it could be
+that you are returning to previous steps in the process. Because of this process, it is hard to define this in this
+project. Therefore, I have chosen comment out the chain-of-thought that one could have in order to f.e. mitigate bias
+or choosing a specific model. So when running `__main__.py` does not result in any output as it will make a model and
+does inference on it without anything to return.
```

Comment:
```suggestion
Building an algorithm is non linear, because of dependencies and cycles in the above mentioned steps. Because of this, it is hard to define this in this in a standard project template. Therefore, I have chosen to make a single Python file and comment out the chain-of-thought that one could have in order to e.g. mitigate bias
or choosing a specific model. So running `__main__.py` will make a model and
does inference on it without anything to return.
```